src/components/table/_table.scss

```diff
@@ -39,9 +88,8 @@
           // `&.sorting`
           cursor: pointer;
 
-          // Up/down sort=null indicator
+          // Up/down `sort=null` indicator
           &::before {
-            display: inline-block;
```

Comment:
Can be removed since the `float` beneath sets the element to `display: block` anyway.

Reply:
That line `display: inline-block;` doesn't exist here ;-)